humble/distribution.yaml

```diff
@@ -7771,6 +7771,21 @@ repositories:
       url: https://github.com/ros-controls/realtime_tools.git
       version: humble
     status: maintained
+  replay_testing:
+    doc:
+      type: git
+      url: https://github.com/polymathrobotics/replay_testing.git
+      version: main
+    release:
+      tags:
+        release: release/humble/{package}/{version}
+      url: https://github.com/polymathrobotics/replay_testing-release.git
+      version: 0.0.1
+    source:
+      type: git
+      url: https://github.com/polymathrobotics/replay_testing.git
+      version: main
+    status: maintained
```

Comment:
i think this is status "developed".  From <https://www.ros.org/reps/rep-0141.html#distribution-file>, there's not actually any info on what they mean but I kind of read it in order of level of activity developed > maintained > unmaintained > end-of-life

Reply:
Forsure-- once I go through some more releases I'll update to maintained